smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/ProcessorMediator.java

```diff
@@ -244,7 +244,7 @@ private void processMethodReturningIndividualPayloadAndConsumingIndividualItem()
         } else {
             this.processor = ReactiveStreams.<Message> builder()
                     .flatMapCompletionStage(managePreProcessingAck())
-                    .map(input -> {
+                    .<Message> map(input -> {
```

Comment:
Any reason why the explicit type is required now?

Reply:
No, it showed up as an error in eclipse before I started working.

> Type mismatch: cannot convert from Processor<Message,Message<Object>> to Processor<Message,Message>

As it builds correctly on the command line, I suspect it's an eclipse issue, though it's possible I have something misconfigured in my workspace. I can take the change out again if you want.

Reply:
oh ok, no problem.